.github/workflows/test.yml

```diff
@@ -1,6 +1,7 @@
 name: test
 
 on:
+  workflow_dispatch:
```

Comment:
For debug purpose?

Reply:
Make it easier to trigger a run on a branch without having to make a PR (can be done in the GitHub GUI if this is added).